package.json

```diff
@@ -5,11 +5,37 @@
   "description": "{{project_name}} frontend assets.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
     "start": "babel-node server.js",
     "build": "NODE_ENV=production webpack -p --progress --colors --config webpack.prod.config.js --bail",
     "lint": "eslint assets"
   },
+  "jest": {
+    "transform": {
+      ".*": "<rootDir>/node_modules/jest-css-modules"
+    },
+    "collectCoverageFrom": [
```

Comment:
If we are adding coverage shouldn't we have a command for coverage?
ie: `"coverage": "jest --coverage"`

Reply:
and add `/coverage/` to gitignore.